ckcp/config.yaml

```
@@ -0,0 +1,26 @@
+# This YAML file allows openshift_dev_setup.sh to fetch values of certain variables.
+# Usage - to be used for overwriting defaults
+
+# CLUSTER_TYPE can only be "openshift" for now.
+CLUSTER_TYPE: openshift
+
+# GIT_URL refers to a git repo to be considered as the source of truth for Argo CD applications.
+GIT_URL: https://github.com/openshift-pipelines/pipeline-service.git
+  
+# GIT_REF refers to the git repo's ref to be considered as the source of truth for Argo CD applications.
+GIT_REF: main
+
+# CR_TO_SYNC is a list of the CRs which need to be synced. 
+CR_TO_SYNC:
+  - deployments.apps
+  - services
+  - ingresses.networking.k8s.io
+  - pipelines.tekton.dev
+  - pipelineruns.tekton.dev
+  - runs.tekton.dev
+  - tasks.tekton.dev
+  - networkpolicies.networking.k8s.io
+  
+# Applications to be deployed on the cluster
+APPS:
+  - pipeline_service # pipeline_service sets up the Pipeline Service on the cluster. 
```

```suggestion
  - pipeline_service # pipeline_service sets up the Pipeline Service on the cluster.
```
Remove trailing spaces